app/page.tsx

```diff
@@ -1,21 +1,25 @@
-import DynamicBadge from "@/components/DynamicBadge";
+import DynamicBadge from "@/components/Badge/DynamicBadge";
 import { isMobileDevice } from "@/utils/isMobileDevice";
-import StaticBadge from "@/components/StaticBadge";
+import StaticBadge from "@/components/Badge/StaticBadge";
+import Experience from "@/components/Experience";
+import Projects from "@/components/Projects";
 
 export const revalidate = 1200;
 
 export default function Home() {
   return (
     <>
       {isMobileDevice() ? (
-        <main className="flex w-full justify-center">
+        <main id="home" className="flex w-full justify-center">
```

Comment:
Can this be set to / or something else?

Reply:
Leave as home for now

Reply:
Although it does result in an issue where clicking on the / doesn't scroll all the way to the top.